README.md

```diff
@@ -1,34 +1,59 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+[![screenshot of the pirates website](.github/readme/screenshot-frontpage.jpg)](https://pirates-ophase.de)
 
-## Getting Started
+# pirates-ophase.de 🏴‍☠️
 
-First, run the development server:
+Yohoho!
+This repository contains the source code for the
+[Pirates website](https://pirates-ophase.de/).
+The website's main purpose is to inform freshmen about our student group and
+what we're doing with them in the o-week.
```

Comment:
'what we got planned for the o-week' would be better

'what we're doing with them' sound pretty threatening